test/parsing-test.js

```diff
@@ -30,6 +30,18 @@ describe('Messages parsing', function () {
         sendGst.restore()
         getRoom.restore()
     })
+    it('return +todo gdocs url', function () {
+      return room.user.say('mikanebu', "bot which todos gdocs").then(function () {
+        assert.equal(room.messages[1][1].substr(0, 14), "@mikanebu http")
```

Comment:
Why are we testing that first 14 letters should be those? why it responds with `@mikanebu http...` - I mean why there is `@Mikanebu` in the message -  what is the actual response?